concept/type/owns.feature

```diff
@@ -0,0 +1,3952 @@
+    When put attribute type: custom-attribute
+    When attribute(custom-attribute) set value-type: <value-type>
+    When entity(player) set owns: custom-attribute
+    Then entity(player) get owns(custom-attribute) set annotation: @values(<args>); fails
+    Then entity(player) get owns(custom-attribute) get annotations is empty
+    When transaction commits
+    When connection open read transaction for database: typedb
+    Then entity(player) get owns(custom-attribute) get annotations is empty
+    Examples:
+      | value-type | args                            |
+      | long       | 0.1                             |
+      | long       | "string"                        |
+      | long       | true                            |
+      | long       | 2024-06-04                      |
+      | long       | 2024-06-04+0010                 |
+      | double     | "string"                        |
+      | double     | true                            |
+      | double     | 2024-06-04                      |
+      | double     | 2024-06-04+0010                 |
+      | decimal    | "string"                        |
+      | decimal    | true                            |
+      | decimal    | 2024-06-04                      |
+      | decimal    | 2024-06-04+0010                 |
+      | string     | 123                             |
+      | string     | true                            |
+      | string     | 2024-06-04                      |
+      | string     | 2024-06-04+0010                 |
+      | string     | 'notstring'                     |
+      | string     | ""                              |
+      | boolean    | 123                             |
+      | boolean    | "string"                        |
+      | boolean    | 2024-06-04                      |
+      | boolean    | 2024-06-04+0010                 |
+      | boolean    | truefalse                       |
+      | datetime   | 123                             |
+      | datetime   | "string"                        |
+      | datetime   | true                            |
+      | datetime   | 2024-06-04+0010                 |
+      | datetimetz | 123                             |
+      | datetimetz | "string"                        |
+      | datetimetz | true                            |
+      | datetimetz | 2024-06-04                      |
+      | datetimetz | 2024-06-04 NotRealTimeZone/Zone |
+      | duration   | 123                             |
+      | duration   | "string"                        |
+      | duration   | true                            |
+      | duration   | 2024-06-04                      |
+      | duration   | 2024-06-04+0100                 |
+      | duration   | 1Y                              |
+      | duration   | year                            |
+
+    # TODO: Maybe we allow it, then change the test considering the expected behavior
```

Comment:
?

Reply:
I think we should only allow 1 annotation of any type anywhere, for simplicity